### DISCUSSION TITLE
Is there a way to invert only one part of a logo?

### DISCUSSION THREAD

**Discussion by Mirm0n** [darkreader/darkreader#9832] [Q&A]
I'm trying to invert the logo here on monday.com

The issue is that it inverts the whole logo, including the icon.

Obviously the icon shouldn't be inverted, only the text. (because now it's the wrong colors)

This is probably impossible, but is there a way to only invert the text? 

Thank you.

![logo monday](https://user-images.githubusercontent.com/21056509/190526354-306e6a7a-9e7c-466d-b916-22f1c76cee0b.png)
![Monday logo 2](https://user-images.githubusercontent.com/21056509/190526356-6dab0b57-9d25-48f8-8387-3dc24f17b457.png)

**Comment by Gusted** [comment]
That's not possible to accomplish via CSS.
